packages/alfa-rules/src/sia-er62/rule.ts

```diff
@@ -38,6 +38,12 @@ const { isText } = Text;
 const { or, not, test } = Predicate;
 const { and } = Refinement;
 
+const hasNonLinkTextCache = Cache.empty<Element, boolean>();
```

Comment:
This cache is only used for one function, so it's better to declare it just before the function (keep the info local).

I'm also realising that `hasNonLinkText` is device dependant, so this cache should be 🙈 not breaking anything since we don't run test with different devices yet, we'll fix that later…